toqito/matrix_ops/tensor_comb.py

```diff
@@ -7,7 +7,11 @@
 from toqito.matrix_ops import to_density_matrix
 
 
-def tensor_comb(states: list[np.ndarray], k: int) -> dict:
+def tensor_comb(
+    k: int,
+    states: list[np.ndarray],
+    mode: str = "non-injective", density_matrix: bool = True,
+    ) -> dict[tuple[int, ...], np.ndarray]:
```

Comment:
This still seems like it's formatting weird:

```suggestion
def tensor_comb(
    k: int,
    states: list[np.ndarray],
    mode: str = "non-injective",
    density_matrix: bool = True,
) -> dict[tuple[int, ...], np.ndarray]:
```